src/components/new-safe/create/logic/index.ts

```diff
@@ -263,3 +265,52 @@ export const getRedirect = (
   const appendChar = hasQueryParams ? '&' : '?'
   return redirectUrl + `${appendChar}safe=${address}`
 }
+
+export const createNewSafeViaRelayer = async (
```

Comment:
Is it possible to use the core-sdk for part of this function i.e. to get the call data?

Reply:
I've checked it but the calls to `encodeSetupCallData` are private that's why I had to call it this way.